pkg/sdk/reconciler/reconciler.go

```diff
@@ -566,7 +567,7 @@ func (r *Reconciler) WatchResourceTypes(resources ...client.Object) error {
 		predicates := []predicate.Predicate{sdk.NewIgnoreLeaderElectionPredicate()}
 
 		if err := r.controller.Watch(source.Kind(r.getCache(), resource), eventHandler, predicates...); err != nil {
-			if meta.IsNoMatchError(err) {
+			if meta.IsNoMatchError(err) || strings.Contains(err.Error(), "failed to find API group") {
```

Comment:
+1

Reply:
I would love if we could not do this :( https://github.com/kubernetes-sigs/controller-runtime/pull/2425#issuecomment-1725606042